web/base_viewer.js

```diff
@@ -149,10 +149,10 @@ class BaseViewer {
       PDFJSDev.test("!PRODUCTION || GENERIC")
     ) {
       if (
-        !(
-          this.container instanceof HTMLDivElement &&
-          this.viewer instanceof HTMLDivElement
-        )
+        !this.container ||
+        !/div/i.test(this.container.tagName) ||
+        !this.viewer ||
+        !/div/i.test(this.viewer.tagName)
```

Comment:
No regular expressions, please!

```suggestion
        !(
          this.container && this.container.tagName.toUpperCase() === "DIV") &&
          this.viewer && this.viewer.tagName.toUpperCase() === "DIV")
         )
```